telemetry.go

```
@@ -262,3 +266,12 @@ func metricViews() []*view.View {
 		drainLengthView,
 	}
 }
+
+func registerMetricViews() error {
+	var registerErr error
+	registerOnce.Do(func() {
```

Is this good enough? Don't we need to add a metric tag that denotes the org?

---

It is good enough to silence the error. I'm not sure if ingest team will internally use these metrics. If they do, we can add support for it later.

---

Thanks owais. I think this is the pragmatic solution for the moment.